SDK Showcase/SDK Showcase/Views/ContentView.swift

```diff
@@ -88,16 +88,20 @@ struct ContentView: View {
             SheetViewForQRScanner()
         }
         .onChange(of: appstate.registeredUsers) {
-            guard category.type == .userAuthentication else { return }
-            
-            category.selection = sdkInteractor.userAuthenticatorOptionNames.map { Selection(name: $0) }
+            updateUsersSelection()
         }
         .onChange(of: appstate.system.enrollmentState) {
             updateMobileAuthenticationCategorySelection()
         }
-        .onAppear() {
+        .onChange(of: appstate.pendingTransactions) {
```

Comment:
I'm not sure. Inside `updateMobileAuthenticationCategorySelection` we are checking `category.type == .mobileAuthentication, appstate.system.enrollmentState != EnrollmentState.unenrolled` so until we are not already on `mobileAuthentication` page and, in your commit, `appstate.pendingTransactions` not changed, than `mobile authentication` button will remain inactive. So TLDR if you are open the app and mobile auth is enrolled, go to Mobile Authentication and the button (selection) should be active. Have you checked that scenario?